docs/book/advanced-guide/practical/switching-orchestration.md

```diff
@@ -2,4 +2,15 @@
 description: Switching orchestrators to run in the cloud
 ---
 
-- New: Mirror according to ZenBytes
+With Stack Recipes, you have everything you need to deploy production-grade
+cloud stacks. Switching stacks is the product of a single CLI command:
```

Comment:
This was supposed to be more about switching orchestration - i would add a lesson here which makes them write a pipeline, and run it on airflow with screenshots